.gitlab-ci.yml

```diff
@@ -46,53 +45,51 @@ variables:
     - cd $DOCKERFILE_DIR
     # - docker pull $CONTAINER_IMAGE:$CONTAINER_TAG || true
     # - docker build --cache-from $CONTAINER_IMAGE:$CONTAINER_TAG --tag $CONTAINER_IMAGE:$CI_BUILD_REF --tag $CONTAINER_IMAGE:$CONTAINER_TAG .
-    - |
-      if [ "${CONTAINER_DATE_TAG}" ]
-      then
-        docker build --no-cache --tag $CONTAINER_IMAGE:$CONTAINER_DATE_TAG --tag $CONTAINER_IMAGE:$CONTAINER_TAG .
-        docker push $CONTAINER_IMAGE:$CONTAINER_DATE_TAG
-      else
-        docker build --no-cache --tag $CONTAINER_IMAGE:$CONTAINER_TAG .
-      fi
+    - export CONTAINER_DATE_TAG="${CI_COMMIT_SHORT_SHA}-$(date +%Y%m%d)"
+    - docker build --no-cache
+        --build-arg VCS_REF="${CI_COMMIT_SHORT_SHA}"
+        --build-arg BUILD_DATE="$(date +%Y%m%d)"
+        --tag $CONTAINER_IMAGE:$CONTAINER_DATE_TAG
+        --tag $CONTAINER_IMAGE:$CONTAINER_TAG .
```

Comment:
* '\\' this one

Reply:
but either way works for me.

Reply:
Sorry, didn't understand you here, elaborate please.

Reply:
I thought you removed it because you didn't see a need for it for the rust build images. For the `kubetools` image it can become handy as the version tag might not change as it is bound to the included helm version, so it is not  possible to refer to a previous image anymore - that's why i added this tag.

Reply:
no, I'm OK with tagging everything as many times as we want, it does not consume disk space.